extensions/sql-bindings/src/services/azureFunctionsService.ts

```diff
@@ -26,6 +26,8 @@ export async function createAzureFunction(node?: ITreeNodeInfo): Promise<void> {
 	let connectionInfo: IConnectionInfo | undefined;
 	let isCreateNewProject: boolean = false;
 	let newFunctionFileObject: azureFunctionsUtils.IFileFunctionObject | undefined;
+	let connectionURI: string = '';
+	let listDatabases: string[] | undefined;
```

Comment:
Why is this up here? You should only need it for the quickpick so just declare it down on line 142